packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -269,21 +267,23 @@ export class SingleThreadedWallet
     }
   }
 
-  async challenge(challengeState: State): Promise<SingleChannelOutput> {
-    const channelId = calculateChannelId(challengeState);
+  async challenge(channelId: string): Promise<SingleChannelOutput> {
     const response = WalletResponse.initialize();
 
     await this.knex.transaction(async tx => {
       const channel = await this.store.getChannel(channelId, tx);
       if (!channel) {
         throw new Error(`No channel found for channel id ${channelId}`);
       }
+      if (!channel.isLedger) {
```

Comment:
Maybe this also needs a `CHALLENGING_V0` comment?

Reply:
Done in 26da120